scripts/sentiment_analysis/process_data.py

```diff
@@ -0,0 +1,138 @@
+# coding: utf-8
+
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Load various datasets."""
+
+from functools import partial
+import multiprocessing as mp
+import re
+import time
+
+import gluonnlp as nlp
+from mxnet import nd, gluon
+
+def _load_file(data_name):
+    if data_name == 'MR':
+        train_dataset = nlp.data.MR(root='data/mr')
+        output_size = 2
+        return train_dataset, output_size
+    elif data_name == 'SST-1':
+        train_dataset, test_dataset = [nlp.data.SST_1(root='data/sst-1', segment=segment)
+                                       for segment in ('train', 'test')]
+        output_size = 5
+        return train_dataset, test_dataset, output_size
+    elif data_name == 'SST-2':
+        train_dataset, test_dataset = [nlp.data.SST_2(root='data/sst-2', segment=segment)
+                                       for segment in ('train', 'test')]
+        output_size = 2
+        return train_dataset, test_dataset, output_size
+    elif data_name == 'Subj':
+        train_dataset = nlp.data.SUBJ(root='data/Subj')
+        output_size = 2
+        return train_dataset, output_size
+    else:
+        train_dataset, test_dataset = [nlp.data.TREC(root='data/trec', segment=segment)
+                                       for segment in ('train', 'test')]
+        output_size = 6
+        return train_dataset, test_dataset, output_size
+
+def _clean_str(string, data_name):
```

Comment:
Can you add a comment above this method as to what the cleaning does for SST1/SST2 datasets and what cleaning is required for other datasets ?
like for ex: it replaces escape characters etc.

Reply:
Since re.sub has a clear documentation, it may be verbose to comment 10+ operations